subsys/mpsl/Kconfig

```diff
@@ -18,4 +19,28 @@ config MPSL_SIGNAL_STACK_SIZE
 	  Size of the signal handler thread stack, used to process lower
 	  priority signals in MPSL.
 
+choice CLOCK_CONTROL_NRF_SOURCE
+	prompt "32 KHz clock source"
+	default CLOCK_CONTROL_NRF_K32SRC_XTAL
+
+config CLOCK_CONTROL_NRF_K32SRC_RC
+	bool "RC Oscillator"
+
+config CLOCK_CONTROL_NRF_K32SRC_XTAL
+	bool "Crystal Oscillator"
+
+config CLOCK_CONTROL_NRF_K32SRC_SYNTH
+	bool "Synthesized from HFCLK"
+
+endchoice
+
+config CLOCK_CONTROL_NRF_ACCURACY
```

Comment:
Can we rather make use of `CLOCK_CONTROL_NRF_K32SRC_500PPM`, `CLOCK_CONTROL_NRF_K32SRC_20PPM` and similar instead?

Reply:
Yes, then we would have to have some ifdef logic in mpsl_init.c to translate from those configs to the value. Should I do that? I don't have any strong opinion about this. But could be nice since we already have this for lf clock source selection.

Reply:
I would rather prefer to reuse existing configs than defining new ones

Reply:
- [ ] Use existing CLOCK_CONTROL configs instead of CLOCK_CONTROL_NRF_ACCURACY

Reply:
Now I remember why I did it like this, it's because the CLOCK_CONTROL_NRF_K32SRC_500PPM etc in nrfxlib clock_control Kconfig will be removed for the ble controller without mpsl, as the ble controller no longer needs the accuracy..

Reply:
Those configs will still exist in the zephyr clock config Kconfig, but will not be included because of CLOCK_CONTROL_NRF_FORCE_ALT.